client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -162,6 +161,14 @@ export default memo( function SitesTableRow( { site }: SiteTableRowProps ) {
 		return siteId && hasSiteStatsQueryFailed( state, siteId, statType, query );
 	} );
 
+	const computeDashboardUrl = ( site: SiteExcerptData, isAtomicSite: boolean | null ) => {
+		if ( isAtomicSite && siteDefaultInterface( site ) === 'wp-admin' ) {
```

Comment:
Should this work for simple sites eventually? Could we be checking for `jetpack non atomic || classic view` instead?

```suggestion
		if ( isJetpackNonAtomic || siteDefaultInterface( site ) === 'wp-admin' ) {
```

Reply:
If that's what we want then I think we could just drop the `isAtomicSite` check

```suggestion
		if ( siteDefaultInterface( site ) === 'wp-admin' ) {
```

Reply:
Much better, thank you!

Reply:
Good spot, thanks makes sense to me. Thank you!

Reply:
minor update - https://github.com/Automattic/wp-calypso/pull/87900 - to remove the atomic site check from the grid view as well. And clean up the now unused isAtomic prop in the helper function above.